streaming_providers/torbox/utils.py

```
@@ -55,20 +53,32 @@ def get_video_url_from_torbox(
     )
 
 
+# Yield successive n-sized
+# chunks from l.
+def divide_chunks(l, n):
```

Ruff

Ambiguous variable name: `l`

(E741)